src/java.base/share/classes/jdk/internal/math/DoubleToDecimal.java

```diff
@@ -101,22 +107,14 @@ public final class DoubleToDecimal {
     private static final int MINUS_INF      = 4;
     private static final int NAN            = 5;
 
-    /*
-     * Room for the longer of the forms
-     *     -ddddd.dddddddddddd         H + 2 characters
-     *     -0.00ddddddddddddddddd      H + 5 characters
-     *     -d.ddddddddddddddddE-eee    H + 7 characters
-     * where there are H digits d
-     */
-    public static final int MAX_CHARS = H + 7;
+    private static final JavaLangAccess JLA = SharedSecrets.getJavaLangAccess();
```

Comment:
JLA is defined repeatedly in many places. Is this a good practice?  Can we use it directly, like this:
```java
return SharedSecrets
           .getJavaLangAccess()
           .newStringNoRepl(bytes, StandardCharsets.ISO_8859_1);
```

Reply:
@wenshao I'll take a note for the next commit, thanks.